Experimental/DistributedAuthoritySample/Assets/Scripts/Player/CameraControl.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Unity.Cinemachine;
+using Unity.Netcode;
+
+namespace Unity.Multiplayer.Samples.SocialHub.Player
+{
+    public class CameraControl : MonoBehaviour
+    {
+        [SerializeField]
+        CinemachineCamera freeLookVCam;
+        [SerializeField]
+        InputActionReference rotateActionReference;
+        [SerializeField]
+        InputActionReference lookActionReference;
+        [SerializeField]
+        float _speedMultiplier = 1.0f; // Speed multiplier for camera movement
+
+        private Transform m_FollowTransform;
+        private bool _cameraMovementLock = false;
+        private bool _isRMBPressed = false;
+        private CinemachineOrbitalFollow _orbitalFollow;
+
+        private void Awake()
+        {
+            if (rotateActionReference != null)
+            {
+                rotateActionReference.action.started += OnRotateStarted;
+                rotateActionReference.action.canceled += OnRotateCanceled;
+                rotateActionReference.action.Enable();
+            }
+
+            if (lookActionReference != null)
+            {
+                lookActionReference.action.performed += OnLookPerformed;
+                lookActionReference.action.Enable();
+            }
+
+            _orbitalFollow = freeLookVCam.GetComponent<CinemachineOrbitalFollow>();
+            freeLookVCam.Follow = null;
+        }
+
+        private void OnEnable()
+        {
+            if (rotateActionReference != null && lookActionReference != null)
+            {
+                rotateActionReference?.action.Enable();
+                lookActionReference?.action.Enable();
+            }
+        }
+
+        private void OnDisable()
+        {
+            rotateActionReference?.action.Disable();
+            lookActionReference?.action.Disable();
+        }
+
+        internal void SetTransform(Transform newTransform)
+        {
+            m_FollowTransform = newTransform;
+
+            if (m_FollowTransform != null)
+            {
+                Camera camera = Camera.main;
+                if (camera != null)
+                {
+                    AvatarTransform.SetCameraReference(camera);
+                }
+                else
+                {
+                    Debug.LogWarning("Main Camera not found. Ensure there is a camera tagged as 'Main Camera' in the scene.");
+                }
```

Comment:
This is the bit that's already done inside AvatarTransform, so no need to do this here anymore.